apps/peer_exchange_tester/peer_exchange_tester.nim

```diff
@@ -0,0 +1,86 @@
+import chronos, chronicles, options, os
```

Comment:
I cannot add a comment in the README.md
The README.md should have content explaining the purpose of the tool and how to use it properly to get the maximum profit of it.